src/commcare_cloud/commands/migrations/couchdb.py

```diff
@@ -53,11 +53,22 @@ class MigrateCouchdb(CommandBase):
             - commit: update database docs with new shard allocation
             - clean: remove shard files from hosts where they aren't needed
         """),
+        Argument('--no-stop', action='store_true', help="""
+            When used with migrate, operate on live couchdb cluster without stopping nodes
+        """),
+        Argument('--skip-file-check', action='store_true', help="""
+            When used with commit, do not check for the existence of expected shard files.
+            Only use this if you've verified the files are in place by other means.
```

Comment:
how would you do that?

Reply:
I used `watch -n10 -d cchq <env> run-shell-command 'ls -al /opt/data/couchdb2/{,.}shards/*/*'` or something like that to watch the rsyncs in progress, and it was a simple enough change that I could tell all the files were there

Reply:
It for some reason was taking much longer than made sense and I couldn't get it to complete. So it was that I wanted to skip it, not exactly that it "failed". Not sure if it had to do with one of the dozens of stats that ansible stats collects being really slow or something.

I can see the argument what we ought to fix whatever that issue is rather than making it easy to skip. Let me know what you think

Reply:
OK I optimized the file check for speed and readability and removed this option. Thanks for talking me through it, I'm much happier with this outcome